### PR TITLE
Owner export UI

### PR DESCRIPTION
**Pull request by elgatovital**
Handles #1451

Changes proposed in this pull request:
- Add owner export screens
- Adjust navigation

@gnosis/mobile-devs

### CODE REVIEW

**Comment by biafra23** [comment]
Te passcode will be required in a separate PR, I guess?

**Comment by biafra23** [comment]
When exporting an imported key (no passphrase) the sub title is malformed:
image

File: app/src/main/java/io/gnosis/safe/ui/settings/owner/details/OwnerDetailsFragment.kt [hunk 1]
```diff
@@ -90,6 +89,7 @@ class OwnerDetailsFragment : BaseViewBindingFragment<FragmentOwnerDetailsBinding
 
                         ownerQrCode.setImageBitmap(viewAction.ownerDetails.qrCode)
                         content.animate().alpha(1f).setDuration(1000)
+                        exportButton.isEnabled = viewAction.ownerDetails.exportable
```
Comment (biafra23) [inline app/src/main/java/io/gnosis/safe/ui/settings/owner/details/OwnerDetailsFragment.kt:92]: exportable would be false for hardware wallet backed keys?

**Comment by biafra23** [comment]
The seed export screen looks much more polished than the key export screen. But that's how our design is ATM.

**Comment by elgatovital** [comment]
> Te passcode will be required in a separate PR, I guess?

yes

File: app/src/main/java/io/gnosis/safe/ui/settings/owner/details/OwnerDetailsFragment.kt (hunk 1 above)
Comment (elgatovital) [reply app/src/main/java/io/gnosis/safe/ui/settings/owner/details/OwnerDetailsFragment.kt:92]: yes, there would be no private key or seed phrase stored on device

**Review by biafra23 — APPROVED** [review]

**Pull request merged**